tests/Resources/Page/OCPDashboard/Events/Events.resource

```diff
@@ -0,0 +1,33 @@
+*** Settings ***
+Documentation       Keyword Suites to work with Openshift Events
+
+Library             Collections
+Library             OpenShiftCLI
+
+
+*** Keywords ***
+Get Event Entry From Openshift
+    [Documentation]    Capture openshift event based on namespace
+    ...    and return filtered data
+    [Arguments]    ${namespace}    ${object_kind}    ${expected_event}=2
```

Comment:
` ${expected_event}=2`

This parameter seems tied to your test and not needed in the library itself.  Why don't you just return an empty list if there are no events and check the list length in the test?

Reply:
i disagree, This parameter is not tied to my test it will be useful for other TC in future if we are dealing with events to make sure we get minimum number of event. If i don't get that i think it is good idea to Fail the Keyword.

Reply:
@tarukumar why setting 2 as threshold?

Reply:
It is based on the manifest how the mapping is..Each resource is mapped to namespace as i am filtering based on namespace it is expected that we should get those number of events and it should be done. Creating one more keyword doesn't do anything. I think i would keep here only. Regarding setting the default value i wouldn't set to 0 if i set to 0 it means there is not a single event. It is default so if people have less or more they can do it when they call the keyword

Reply:
I kind of still disagree, but if you think this is the best way to do that let's keep it, no issues :)

Reply:
I have changed it to 1